### DISCUSSION TITLE
快速上手 | NoneBot 文档

### DISCUSSION THREAD

**Discussion by giscus** [KomoriDev/NoneBotX#60] [General]
# 快速上手 | NoneBot 文档

NoneBot2 社区文档

https://nonebotx.netlify.app/guide/hello_hibiscus

**Comment by sunyuxi4** [comment]
诶嘿略略略

**Reply by KomoriDev** [reply]
你小子
<img width=300 src="https://github.com/user-attachments/assets/c4e8533d-7517-48ae-87b6-b9fbb7a68288" alt="" />
